ports/servoshell/egl/ohos.rs

```diff
@@ -362,6 +366,16 @@ pub fn load_url(url: String) -> Undefined {
     call(ServoAction::LoadUrl(url)).expect("Failed to load url");
 }
 
+#[napi]
+pub fn go_back() -> Undefined {
```

Comment:
Question: why do you return `Undefined ` here?

Reply:
The `napi` macro from `napi-rs` maps `Undefined` / `()` in Rust to `undefined` in JS.
I just tested and the macro does seem to support signatures without an explicit return type, so removing it is probably best.

Reply:
Thanks. Indeed, removing it makes sense; 'Undefined' got me confused.